playbooks/common/openshift-cluster/config.yml

```diff
@@ -26,6 +26,21 @@
   tags:
   - always
 
+- name: Setup firewall
+  hosts: oo_all_hosts
+  tags:
+  - always
+  tasks:
+  # This should move to intialize_facts
+  - name: set os_firewall_enabled
+    set_fact:
+      os_firewall_enabled: true
+      os_firewall_use_firewalld: false
+
+  - name: Set proper firewall settings
+    include_role:
+      name: os_firewall
```

Comment:
With the function of the os_firewall role being changed from installing/configuring the firewall AND managing rules, the README should be updated to reflect the new function of just installation/configuration.